README

```diff
@@ -25,5 +26,31 @@
  - make install
 
  Make sure that you do not have libcoap-1-0-dev installed, as the headers might conflict.
+
 ===
+
```

Comment:
Can we use .md format for README, as we do in other projects?

Reply:
@macpijan fixed in new PR